src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -267,6 +269,24 @@ class editIndex extends viewModelBase {
             });
     }
 
+    private fetchCustomAnalyzers() {
+        const db = this.activeDatabase();
+        
+        new getCustomAnalyzersCommand(db, true)
+            .execute()
+            .done((customAnalyzers) => {
+                const analyzerNames = customAnalyzers.map(x => x.Name);
+                this.editedIndex().customAnalyzers(analyzerNames);
```

Comment:
can we introduce method registerCustomAnalyzers on this.editedIndex() class and it can set everything for us? 
Which is: customAnalyzers field, call addCustomAnalyzers on all fields + addCustomAnalyzers on default field?

Reply:
done